<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tortoiseRFID</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_Initialize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>115</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Initialize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Exit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_GetMoney">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Check Account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_AddMoney">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Add Money</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_CutMoney">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Cut Money</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_GetMoney">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>115</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_AddMoney">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>115</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CutMoney">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>115</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
